Misc/NEWS.d/next/Library/2022-11-17-10-02-18.gh-issue-94912.G2aa-E.rst

```diff
@@ -0,0 +1 @@
+Added ``@markcoroutinefunction`` decorator to ``inspect`` module.
```

Comment:
I'd rephrase this using present tense (like most other news entries) and using ```:func:`inspect.markcoroutinefunction` ```.
```suggestion
Add :func:`inspect.markcoroutinefunction` decorator which manually marks
a function as a coroutine for the benefit of :func:`iscoroutinefunction`.
```